scripts/bulk_price_upload/run-bulk-price-upload.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This is new and avoids the need to go onto a pod directly to import the prices.

Reply:
nice!